lib/iris/_lazy_data.py

```diff
@@ -42,17 +42,23 @@ def _iris_dask_defaults():
     all available CPUs.
 
     .. note::
+
         We only want Iris to set dask options in the case where doing so will
         not change user-specified options that have already been set.
 
     """
-    if 'pool' not in dask.context._globals and \
-            'get' not in dask.context._globals:
-        dask.set_options(get=dget_sync)
-
-
-# Run this at import time to set dask options for Iris.
-_iris_dask_defaults()
+    dask_opts = {}
+    dask_globals = getattr(dask.context, '_globals')
+    if dask_globals is not None:
```

Comment:
So if dask change their ```dask.context._globals``` storage location, we let dask give us the default engine. Is that what is intended?